packages/thrift-integration/src/client/connections/HttpConnection.spec.ts

```diff
@@ -189,33 +190,33 @@ describe('HttpConnection', () => {
                 new Calculator.Client(badConnection)
 
             return badClient.add(5, 7).then(
-                (response: number) => {
+                () => {
                     throw new Error('Should reject with host not found')
                 },
-                (err: any) => {
+                (err: Error) => {
                     expect(err).to.exist()
                 },
             )
         })
 
         it('should use request implementation if provided', async () => {
             let ourRequestWasCalled = false
-            const mockRequest = request.defaults({
-                auth: {
-                    bearer: () => {
-                        ourRequestWasCalled = true
-                        return 'token'
-                    },
-                },
-            })
+            const requestSpy = async (
```

Comment:
Nit - unnecessary async?

Reply:
Yep, good catch - was playing around with this "mock" to fix the tests and forgot to remove that.